### ISSUE TITLE
ReproducibiliTea code of conduct?

### ISSUE THREAD

**Issue by eirini-zormpa** [Radboud-University/osc-nijmegen#56]
curious to see if you think this is useful. Depending on who comes and how touchy the issues we discuss are, this could come in handy. Nothing too radical, I was thinking something along the lines of "don't interrupt people when they talk", "don't monopolise the conversation", "don't use offensive language".

**Comment by KristijanArmeni** [comment]
Good point. I think the broader points such as "don't use offensive language" are in some way covered in OSCN CoC. But I agree that it would be useful to come up with some loose guidelines for participation. And make those visible somewhere. I think in line with this message/initiative:

https://scientistsarehumans.com/

**Comment by eirini-zormpa** [comment]
ooooh I didn't know about this! their manifesto looks awesome :) I'll take a look later today and see if I can start with a few very general guidelines
